### ISSUE TITLE
UX - Letzte Aktivität filterbar machen nach Aktivitätstyp

### ISSUE THREAD

**Issue by juebaum** [stadtluzern/decidim-ocl#44]

**Comment by juebaum** [comment]
![letzte_aktivitaet](https://user-images.githubusercontent.com/78081549/106723201-25ce1880-6607-11eb-8e63-1798101ade6a.png)

**Comment by Kagemaru** [comment]
`Last Activities` wurde im Moment von PROD entfernt aufgrund von Performance Problemen.

@innosmith Sollen wir dies nach dem Upgrade auf 0.24 anschauen, da dies momentan keine Auswirkung hat?

**Comment by innosmith** [comment]
Ja, auf 0.24 warten wegen der grundsätzlichen Performance-Verbesserungen.

**Comment by innosmith** [comment]
Ohne Klick muss es möglich sein, dass zuerst letzte Blog-Beiträge angezeigt werden können. 

Optional gefolgt von weiteren noch zu definierende Kategorien wie letzte Umfragen, ...

**Comment by carlobeltrame** [comment]
Grobschätzung: Für ersten Baustein 0.5 - 1 PT, für weitere Bausteine je 0.5 PT

**Comment by carlobeltrame** [comment]
Ich habe "Neueste Blogbeiträge" sowie "Neueste Umfragen" implementiert und auf Integration aufgeschaltet. Letzteres funktioniert aber nicht wirklich, weil in den "last_activities" von decidim nie Umfragen auftauchen. Möglicherweise ein Bug in decidim, oder ich verstehe nicht, wie eine Umfrage konfiguriert werden müsste, damit sie dort auftaucht.
Weitere Bausteine wären sehr einfach umzusetzen, solange sie in der Filter-Liste hier links bereits vorhanden sind: https://test.dialogluzern.ch/last_activities

**Comment by innosmith** [comment]
@carlobeltrame : sind die neuesten Blogbeiträge auf der Startseite wirklich freigeschaltet? Ich sehe noch alle letzten Aktivitäten. Bei den aktiven Organisationen auf der Startseite sieht man keine Bilder mehr. Bei den Veranstaltungen ist der Kartenlayer leer. Letzteres hat vielleicht mit dem laufenden Swisstopo-Update zu tun.

**Comment by carlobeltrame** [comment]
Ich habe nicht den "letzte Aktivitäten" Block überschrieben oder abgeändert, sondern zwei neue Homepage-Blöcke gemacht, die ihr selber am gewünschten Ort platzieren könnt, via Einstellungen -> Startseite. Ich habs jetzt kurz reingezogen über den "letzte Aktivitäten" Block.

Die aktiven Organisationen die gerade sichtbar sind haben soweit ich sehe auf der Testumgebung schlicht kein Bild gesetzt. Ich habe kurz beim Luzerner Tauschnetz ein Bild hochgeladen (und inzwischen wieder entfernt), und es wurde korrekt angezeigt.

Die Kartenanzeigen sind wie du korrekt angenommen hast noch leer wegen einem Problem in #121. Da arbeite ich heute noch dran.

**Comment by innosmith** [comment]
@carlobeltrame : sehr gute Lösung mit den Blog-Beiträgen als sep. Startseiten-Komponente! -> ready for production.

**Comment by larsUE** [comment]
Close hier weil dies eine Pre-Redesign Funktionseinschränkung betrifft, die nun nicht mehr auftritt.
